packages/vats/decentral-loadgen-config.json

```diff
@@ -0,0 +1,65 @@
+{
+  "bootstrap": "bootstrap",
+  "vats": {
+    "bootstrap": {
+      "sourceSpec": "./src/core/boot.js",
+      "parameters": {
+        "governanceActions": true,
+        "makeClientManager": {
+          "template": {
+            "agoricNames": true,
+            "bank": true,
+            "namesByAddress": true,
+            "myAddressNameAdmin": true,
+            "board": true,
+            "faucet": true,
+            "zoe": true,
+            "vaultFactoryCreatorFacet": true
+          }
```

Comment:
@michaelfig the keys of this template are supposed to be the things that the client manager waits for before resolving the chainbundle. Is this obsolete as of the notifier-based protocol for `home` properties?

Can the loadgen get that notifier?

Reply:
> @michaelfig the keys of this template are supposed to be the things that the client manager waits for before resolving the chainbundle. Is this obsolete as of the notifier-based protocol for `home` properties?

It's mostly orthogonal.  Those things are the items that must be present before the *first* time the notifier fires.

> Can the loadgen get that notifier?

Not currently.  That would be a good change (always include the notifier as one of the properties on `agoric`).

Until then, the template ensures deploy scripts can have access to those objects.  Without the template and without the notifier, there is only polling.  So let's always ensure one of those is in force.